server/router_utils.go

```diff
@@ -141,14 +141,29 @@ func readClusterName(writer http.ResponseWriter, request *http.Request) (types.C
 
 // readOrganizationID retrieves organization id from request
 // if it's not possible, it writes http error to the writer and returns error
-func readOrganizationID(writer http.ResponseWriter, request *http.Request) (types.OrgID, error) {
+func readOrganizationID(writer http.ResponseWriter, request *http.Request, auth bool) (types.OrgID, error) {
 	organizationID, err := getRouterPositiveIntParam(request, "organization")
 	if err != nil {
 		handleOrgIDError(writer, err)
 		return 0, err
 	}
+	err = checkPermissions(writer, request, types.OrgID(organizationID), auth)
```

Comment:
maybe it's better to check it somwhere else like in authorization middleware? Because router_utils is supposed to be only for parsing parameters from route.

Reply:
Actually no, because auth middleware is not processing URL, so it should do route handler

Reply:
it has everything sent in request actually, but ok, let's keep it as is.